rpcs3/rpcs3qt/game_list.cpp

```diff
@@ -45,3 +45,13 @@ void game_list::leaveEvent(QEvent */*event*/)
 		m_last_hover_item = nullptr;
 	}
 }
+
+void game_list::FocusAndSelectFistEntryIfNoneIs()
```

Comment:
Is this intended to say FocusAndSelectFirstEntryIfNoneIs?
At least there is consistency with Fist. Not a big issue but easy enough to clean up.

Reply:
Thanks, fixed.